BUILD

```diff
@@ -540,12 +540,100 @@ internal_gen_well_known_protos_java(
 )
 
 java_library(
+    name = "protobuf_java_lite",
```

Comment:
if the entire code is in `java/core/src/main/java/com/google/protobuf/`, perhaps put a BUILD file there?